test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -4,13 +4,14 @@
 import static org.junit.Assert.assertTrue;
 
 import java.util.Arrays;
+import java.util.HashMap;
```

Comment:
Good try in getting the naming convention right 👍 

Note the naming convention for test methods (the first part is feature/method under test), second is the testing scenario and the third is the expected behaviour. 
E.g. isAnyNull_emptyList_falseReturned
Part 2 and 3 of the test name can be removed to indicate an increased test scope